agent/rpc_from.go

```diff
@@ -99,6 +99,8 @@ func (a *orbAgent) handleGroupRPCFromCore(client mqtt.Client, message mqtt.Messa
 			return
 		}
 		a.handleAgentPolicies(r.Payload, r.FullList)
+		a.logger.Debug("received agent policies, setting success in request")
```

Comment:
this messages a little confusing for me, can we make it "received agent policies, marking success"